src/test/java/se/diabol/jenkins/pipeline/PipelineVersionContributorTest.java

```diff
@@ -150,18 +145,15 @@ public void testVersionContributorConfiguredManualTrigger() throws Exception {
         jenkins.waitUntilNoActivity();
 
         assertNotNull(secondProject.getLastBuild());
-
     }
 
     @Test
     public void testIsApplicable() throws Exception {
         PipelineVersionContributor.DescriptorImpl d = new PipelineVersionContributor.DescriptorImpl();
         assertTrue(d.isApplicable(jenkins.createFreeStyleProject("a")));
-
     }
 
-    @Test
-    @Bug(21070)
+    @Test // JENKINS-21070)
```

Comment:
In Spock, for instance, `@Issue` annotation can be detected and presented (as a link) in BDD reports. I don't know if Jenkins uses it for any automatized activities.